pkg/generate/app/imagestreamlookup.go

```diff
@@ -33,9 +33,11 @@ func (r ImageStreamSearcher) Search(terms ...string) (ComponentMatches, error) {
 		if len(ref.Namespace) != 0 {
 			namespaces = []string{ref.Namespace}
 		}
+		followTag := false
 		searchTag := ref.Tag
 		if len(searchTag) == 0 {
 			searchTag = imageapi.DefaultImageTag
+			followTag = true
```

Comment:
comment this.  on what basis did we decide to set followTag to true?  what user intent are we trying to execute on here?  ("user didn't specify a specific tag, therefore we are going to try to match to....")